Containers/clamav/Dockerfile

```diff
@@ -1,28 +1,21 @@
 # syntax=docker/dockerfile:latest
-# Probably from this file: https://github.com/Cisco-Talos/clamav-docker/blob/main/clamav/1.3/alpine/Dockerfile
-FROM clamav/clamav:1.4.2-28
+FROM alpine:3.21.3
 
-COPY clamav.conf /clamav.conf
-COPY --chmod=775 start.script /start.script
+RUN apk upgrade --no-cache -a && \
+    apk add --no-cache tzdata clamav && \
+    mkdir /run/clamav && \
+    chmod 777 -R /run/clamav /var/log/clamav && \
+    sed -i "s|#\?MaxDirectoryRecursion.*|MaxDirectoryRecursion 30|g" /etc/clamav/clamd.conf && \
+    sed -i "s|#\?MaxFileSize.*|MaxFileSize 2G|g" /etc/clamav/clamd.conf && \
```

Comment:
```suggestion
    sed -i "s|#\?MaxFileSize.*|MaxFileSize aio-placeholder|g" /etc/clamav/clamd.conf && \
```

Reply:
the limit is 2G: https://github.com/Cisco-Talos/clamav/blob/492e505070499a5e7eb60afc89fdec42e55b1d8d/etc/clamd.conf.sample#L585

Reply:
ok